build.gradle

```diff
@@ -952,6 +952,36 @@ project(':iceberg-snowflake') {
 }
 
 project(':iceberg-open-api') {
+  apply plugin: 'java-test-fixtures'
```

Comment:
`RESTCompatibilityKitSuite` is currently under a `test` source directory? How is it published (sorry, I could not find even older classes from that module in Maven Central)?

Reply:
I don't believe there any prior artifacts, so this would be new.  The test fixtures can be depended upon by other projects, but that is just the test rest server implementation.  I think we can publish the tests artifacts as well (we do this for `iceberg-data` I believe.

Reply:
@dimas-b it looks like we originally blocked publishing artifacts for this subproject, but I've updated it now to publish both the tests and testFixtures jars.

Reply:
Thx. I guess the best path forward for now is to merge, release and try it out in downstream projects.